resources/zib/zib-MedicationAdministration2.xml

```diff
@@ -0,0 +1,398 @@
+      </type>
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.23170" />
+        <comment value="MedicationAgreement" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.note">
+      <path value="MedicationAdministration.note" />
+      <max value="1" />
+    </element>
+    <element id="MedicationAdministration.note.text">
+      <path value="MedicationAdministration.note.text" />
+      <short value="Comment" />
+      <definition value="Comments on administering the medication." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.21337" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.dosage.route">
+      <path value="MedicationAdministration.dosage.route" />
+      <short value="RouteOfAdministration" />
+      <definition value="The route through which the medication is administered (oral, nasal, intravenous,...)." />
+      <alias value="Toedieningsweg" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.9.13.2--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.21195" />
+        <comment value="RouteOfAdministration" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.dosage.dose">
+      <path value="MedicationAdministration.dosage.dose" />
+      <short value="AdministeredAmount" />
+      <definition value="Amount of the administered product." />
+      <alias value="ToegediendeHoeveelheid" />
+      <type>
+        <code value="Quantity" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/pattern-GstdSimpleQuantity" />
+      </type>
+      <mapping>
+        <identity value="zib-medicationadministration2-v1.1.1-2020EN" />
+        <map value="NL-CM:9.13.21194" />
+        <comment value="AdministeredAmount" />
+      </mapping>
+    </element>
+    <element id="MedicationAdministration.dosage.rate[x]">
+      <path value="MedicationAdministration.dosage.rate[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="MedicationAdministration.dosage.rate[x].extension">
+      <path value="MedicationAdministration.dosage.rate[x].extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="0" />
+    </element>
+    <element id="MedicationAdministration.dosage.rate[x].extension:range">

```

Comment:
@ArdonToonstra the zib compliance tool trips on this piece of code. Shouldn't the extension be in the `:rateRange` slice?

Reply:
Didn't get it to work. Removed it all. Forge can't really handle it as well. I think I was bending FHIR to it's limits.